packages/blockchain-wallet-v4-frontend/src/data/auth/sagas.js

```diff
@@ -111,7 +111,7 @@ export default ({ api, coreSagas }) => {
             yield call(coreSagas.wallet.fetchWalletSaga, { guid, session, password })
             yield call(loginRoutineSaga, mobileLogin)
           } catch (e) {
-            yield put(actions.alerts.displayError(error || 'Error logging into your wallet'))
+            yield put(actions.alerts.displayError('Error logging into your wallet'))
```

Comment:
Can we console log the actual error as well?

Reply:
Yes I've just fixed this, displaying the error.message